.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: false
+sudo: required
```

Comment:
`sudo` is deprecated now.

Reply:
We were using it for the coverage job before, and this just carries that over. I have no idea if it is actually required or not.